code/current_release.R

```diff
@@ -2,4 +2,4 @@ release <- "20Q1"
 achilles_url <- "https://ndownloader.figshare.com/files/21521910" #achilles_gene_effect.csv
 ccle_url <- "https://ndownloader.figshare.com/files/21521940" #CCLE_expression.csv
 cclemeta_url <- "https://ndownloader.figshare.com/files/21522000" #sample_info.csv
-na_cutoff <- 631 #~5% FDR
\ No newline at end of file
+na_cutoff <- readRDS(file = here::here("data", paste0(release, "_na_cutoff.Rds"))) #~5% FDR
```

Comment:
I think you have a chicken and egg problem here.  This file is sourced by `find_threadshold.R` which will create the 20Q2_na_cutoff.Rds file but fails with `No such file or directory`.
One possible solution is to only set na_cutoff if the file exists:
```suggestion
na_cutoff_file = here::here("data", paste0(release, "_na_cutoff.Rds"))
if (file.exists(na_cutoff_file)) {
  na_cutoff <- readRDS(file = na_cutoff_file) #~5% FDR  
}
```